aui.views/src/AUI/Platform/win32/AWindowsImpl.cpp

```diff
@@ -519,3 +519,8 @@ void AWindow::allowDragNDrop() {
     auto r = RegisterDragDrop(mHandle, new DropTarget(this));
     assert(r == S_OK);
 }
+
+void AWindow::focusNextView() {
+    setFocusNextViewOnTab(false);
+    ABaseWindow::focusNextView();
+}
```

Comment:
Are you sure these lines should be in the framework, not in the client application (MagicSea)?